package.json

```
@@ -51,6 +51,7 @@
     "@storybook/mdx2-csf": "0.0.3",
     "@storybook/react": "6.5.6",
     "@storybook/testing-library": "0.0.11",
+    "@storybook/theming": "^6.5.6",
```

Please make sure to always pin dependencies: 
```suggestion
    "@storybook/theming": "6.5.6",
```